tests/testthat/test-object_usage_linter.R

```diff
@@ -545,7 +545,8 @@ test_that("fallback works", {
     "),
     list(
       message = rex::rex("no visible global function definition for ", anything, "non_existing_assign<-"),
-      column_number = 6L
+      line_number = 2L,
+      column_number = 3L
```

Comment:
Added benefit of the improved lint locations is we actually find a better way to place this one :)